src/Backend/Modules/Pages/Actions/Add.php

```diff
@@ -550,12 +554,33 @@ private function validateForm(): void
         }
     }
 
-    private function getImage(bool $allowImage): ?string
+    private function getImage(bool $allowImage, ?string $defaultImage): ?string
     {
-        if (!$allowImage || !$this->form->getField('image')->isFilled()) {
+        if (!$allowImage) {
             return null;
         }
 
+        if ($defaultImage === null && !$this->form->getField('image')->isFilled()) {
```

Comment:
why don't you group those if statements since they all return null?

Reply:
Fixed 40e3308c2